src/config/workload.rs

```diff
@@ -359,6 +359,11 @@ pub enum Verb {
     /// * RESP: `RPOP`
     #[serde(alias = "rpop")]
     ListPopBack,
+    /// Removes all elements with given value from the list
```

Comment:
Let's note that this is not currently implemented for RESP.

Reply:
updated the comment.